src/titanium-company-selector.ts

```diff
@@ -92,73 +102,89 @@ export class TitaniumCompanySelectorElement extends PolymerElement {
         {bubbles: true, composed: true, detail: {message: error}}));
   }
 
+  private setCompanyId(id: number|null) {
+    this.dispatchEvent(new CustomEvent(
+        'companyid-changed', {composed: true, detail: {value: id}}));
```

Comment:
has to be  `company-id-changed` for polymer compatibility

Reply:
You are correct. And that is going to be confusing. This element is now going to look for companyid to be passed in, but is going to be passing out via company-id which means two-way binding on this property isn't going to work.

Can I ask why we've suddenly decided that hyphen-case is bad?